Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
-name = "someip-parser"
-version = "0.2.0"
+name = "someip-header"
```

Comment:
A naming like "someip-proto" or "someip-messages" would be more accurate for this crate, if it deals with the possible someip messages "on the wire" for SD and RPC (without the RPC specific types).

Reply:
Agree. Possible more candidates:

* `someip-wire`
* `someip-basic`
* `someip-frame`

@marcmo @jens-esr ?

Reply:
I like `someip-message` or `someip-frame`

Reply:
Everyone fine with `someip-message`? Think it's a good fit!